test/Java9andUp/src/org/openj9/test/java9AttachAPI/TargetManager.java

```diff
@@ -0,0 +1,412 @@
+/*******************************************************************************
+ * Copyright (c) 2001, 2017 IBM Corp. and others
```

Comment:
As this is new code the start date should be 2017.

Reply:
Fixed.